aws/structure_test.go

```diff
@@ -1199,70 +1199,6 @@ func TestFlattenPolicyAttributes(t *testing.T) {
 	}
 }
 
-func TestNormalizeJsonString(t *testing.T) {
-	var err error
-	var actual string
-
-	// Well formatted and valid.
-	validJson := `{
-   "abc": {
-      "def": 123,
-      "xyz": [
-         {
-            "a": "ホリネズミ"
-         },
-         {
-            "b": "1\\n2"
-         }
-      ]
-   }
-}`
-	expected := `{"abc":{"def":123,"xyz":[{"a":"ホリネズミ"},{"b":"1\\n2"}]}}`
-
-	actual, err = normalizeJsonString(validJson)
-	if err != nil {
-		t.Fatalf("Expected not to throw an error while parsing JSON, but got: %s", err)
-	}
-
-	if actual != expected {
-		t.Fatalf("Got:\n\n%s\n\nExpected:\n\n%s\n", actual, expected)
-	}
-
-	// Well formatted but not valid,
-	// missing closing squre bracket.
-	invalidJson := `{
-   "abc": {
-      "def": 123,
-      "xyz": [
-         {
-            "a": "1"
-         }
-      }
-   }
-}`
-	actual, err = normalizeJsonString(invalidJson)
-	if err == nil {
-		t.Fatalf("Expected to throw an error while parsing JSON, but got: %s", err)
-	}
-
-	// We expect the invalid JSON to be shown back to us again.
-	if actual != invalidJson {
-		t.Fatalf("Got:\n\n%s\n\nExpected:\n\n%s\n", actual, invalidJson)
-	}
-
-	// Verify that it leaves strings alone
-	testString := "2016-07-28t04:07:02z\nsomething else"
-	expected = "2016-07-28t04:07:02z\nsomething else"
-	actual, err = normalizeJsonString(testString)
```

Comment:
Likewise - do you mind PRing this upstream? https://github.com/hashicorp/terraform/blob/master/helper/structure/normalize_json_test.go

Reply:
Thanks for the catch, it has been accepted upstream. Merging!